### PR TITLE
OG-815 Calling RelayProvider.send(...) without gas fees params

### PR DESCRIPTION
**Pull request by shahafn**

### CODE REVIEW

File: packages/cli/src/GsnTestEnvironment.ts [hunk 1]
```diff
@@ -186,7 +186,6 @@ class GsnTestEnvironmentClass {
       reputationManager
     }
     const relayServerParams: Partial<ServerConfigParams> = {
-      coldRestartLogsFromBlock: 1,
```
Comment (drortirosh) [inline packages/cli/src/GsnTestEnvironment.ts:189]: unrelated. this is only because you make a PR out of another PR's branch. 
I suggest to avoid it, as it most cases, you'll need to update code and not assume both PRs will get in, and thus it will also make reviewing it easier.

Comment (shahafn) [reply packages/cli/src/GsnTestEnvironment.ts:189]: Please review the former PR first. After that, I will rebase it and it will be fine.

Comment (drortirosh) [reply packages/cli/src/GsnTestEnvironment.ts:189]: I understand. but PRs that are unrelated are better be based on master, for easier review. they will need "update to master" anyway..

**Review by drortirosh — APPROVED** [review]
